### PR TITLE
soletta: Add Node.js bindings to soletta package

### PR DESCRIPTION
**Pull request by nagineni**
This patch exports the needed variables to build Node.js bindings and add them to the soletta package.

[1] This
[2] ostroproject/meta-ostro#115 (Enable bindings by default)
[3] ostroproject/meta-iot-web#10 (Add soletta-nodejs package to Node.js runtime package group)

This replaces #71 

Signed-off-by: Sudarsana Nagineni sudarsana.nagineni@intel.com

### CODE REVIEW

File: recipes-soletta/soletta/soletta_git.bb [hunk 1]
```diff
@@ -81,7 +86,55 @@ do_configure_prepend() {
...
+           ;;
+       *) echo "unknown architecture"
+          exit 1
+           ;;
+   esac
+
+   # Export needed variables to build Node.js bindings
+   export CFLAGS="$CFLAGS -fPIC"
+   export CXXFLAGS="$CXXFLAGS -fPIC"
+   export NODE_GYP="${STAGING_DIR_NATIVE}/${libdir}/node_modules/npm/bin/node-gyp-bin/node-gyp --arch=${targetArch} ${NODE_GYP_PROXY}"
```
Comment (mythi) [inline recipes-soletta/soletta/soletta_git.bb]: this expects nodejs-native in DEPENDS, right?

Comment (nagineni) [inline recipes-soletta/soletta/soletta_git.bb]: Good catch. Forgot to commit a local change that I made to use TARGET instead of NATIVE here.

**Comment by nagineni** [comment]
@brunobottazzini @mythi PTAL

Test build is green with all the changes[1], [2] and [3]: https://github.com/ostroproject/ostro-os/pull/66

**Comment by mythi** [comment]
+1

**Comment by brunobottazzini** [comment]
+1

**Pull request merged**